Tasks/AzureDevOpsTestPlanV0/automatedTests.ts

```diff
@@ -0,0 +1,53 @@
+import tl = require('azure-pipelines-task-lib/task');
+import { testInvoker } from './automatedTestInvoker'
+import { TestPlanData, getAutomatedTestData } from './getAutomatedTests';
+import { TestCase } from 'azure-devops-node-api/interfaces/TestPlanInterfaces';
+
+
+export async function automatedTestsFlow() {
+
+    let ListOfTestsToBeExecuted: string[] = [];
+
+    console.log(tl.loc('automatedTestsTriggered'));
+    await getFQNsOfAutomatedTestCases()
+        .then((testsToBeExecuted) => {
+            ListOfTestsToBeExecuted = testsToBeExecuted;
+        })
+        .catch((error) => {
+            console.error("Promise rejected:", error);
+        });
+
+    tl.debug("Invoking test execution for tests: " + ListOfTestsToBeExecuted);
```

Comment:
should we have a check here, if ListOfTestsToBeExecuted is empty we skip making this call?

Reply:
added check, the task will fail if there are no automated test cases and exit

Reply:
need to check if task should pass/fail in this case

Reply:
Yes, needn't necessarily fail the task as there might be manual tests which got executed.

Reply:
Updated the condition to fail in automated flow if automated tests are not present